src/train.py

```diff
@@ -0,0 +1,70 @@
+from typing import Dict
+import pandas as pd
+import os
+from joblib import dump
+from model import (
+    bag_of_words,
+    mlb,
+    tf_idf
+)
+from data.preprocessing import read_data
+
+
+def get_corpus_counts(X_train: list[str], y_train: list[str]):
+    tags_counts: Dict[str, int] = {}
+    words_counts: Dict[str, int] = {}
+
+    for sentence in X_train:
+        for word in sentence.split():
+            if word in words_counts:
+                words_counts[word] += 1
+            else:
+                words_counts[word] = 1
+
+    for tags in y_train:
+        for tag in tags:
+            if tag in tags_counts:
+                tags_counts[tag] += 1
+            else:
+                tags_counts[tag] = 1
+
+    return words_counts, tags_counts
+
+
+def main():
+    train = read_data('assets/data/processed/train.tsv')
```

Comment:
Maybe make the first 6 lines of this into a function? Because this code is also used in `preprocessing.py` in the same way

Reply:
Done